app/assets/stylesheets/layout/_default_list.scss

```diff
@@ -1,10 +1,10 @@
-.authors-list {
+.default-list {
```

Comment:
Any suggestion of a better naming here? We're using it on authors index page too.